src/amber/router/flash.cr

```diff
@@ -9,7 +9,10 @@ module Amber
         FlashStore.from_session(flash_content)
       end
 
-      class FlashStore < Hash(String, String)
+      class FlashStore
+        @raw : Hash(String, String)
```

Comment:
The name `raw` doesn't mean much to me, but I'm not sure I can think of a better name. Perhaps `@store`?